src/main/java/uk/ac/ebi/embl/api/validation/check/file/FileValidationCheck.java

```diff
@@ -211,6 +212,37 @@ protected int getAssemblyLevel(ValidationScope scope)
 		return assemblyLevel;
 	}
 
+	public void covid19SeqLengthCheck() throws ValidationEngineException {
+		if (sharedInfo.assemblyType != null && sharedInfo.assemblyType.equalsIgnoreCase(AssemblyType.COVID_19_OUTBREAK.getValue())) {
+			if (sharedInfo.chromosomeNames.size() > 1) {
+				throw new ValidationEngineException("COVID-19 genome submissions must have a single chromosome sequence", ReportErrorType.VALIDATION_ERROR);
+			}
```

Comment:
- We can't assume that there would be only a single chromosome. We have ~100 other cases as well, not many but they still exist. Having single chromosome restriction would be possible but we would need to discuss it first with the ENA helpdesk if this is something we could impose.
```
select chromosome_cnt, contig_cnt, scaffold_cnt, count(1)
from gcs_assembly 
where assembly_type = 'COVID-19 outbreak'
group by  chromosome_cnt, contig_cnt, scaffold_cnt;
```